applications/OptimizationApplication/python_scripts/convergence_criteria/combined_conv_criterion.py

```diff
@@ -0,0 +1,75 @@
+import typing
+import KratosMultiphysics as Kratos
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem import OptimizationProblem
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem_utilities import OptimizationComponentFactory
+from KratosMultiphysics.OptimizationApplication.convergence_criteria.convergence_criterion import ConvergenceCriterion
+
+def Factory(model: Kratos.Model, parameters: Kratos.Parameters, optimization_problem: OptimizationProblem) -> ConvergenceCriterion:
+    if not parameters.Has("settings"):
+        raise RuntimeError(f"CombinedConvCriterion instantiation requires a \"settings\" in parameters [ parameters = {parameters}].")
+    return CombinedConvCriterion(model, parameters["settings"], optimization_problem)
+
+class CombinedConvCriterion(ConvergenceCriterion):
+    @classmethod
+    def GetDefaultParameters(cls):
+        return Kratos.Parameters("""{
+            "operator"                    : "",
```

Comment:
Can we update the list of available operators or add a comment to the existing ones?

Reply:
done :)